src/use-cases/commands/InviterUnUtilisateur.ts

```diff
@@ -16,14 +18,38 @@ export class InviterUnUtilisateur implements CommandHandler<InviterUnUtilisateur
       return 'KO'
     }
     const utilisateurACreer = command.role
-      ? Utilisateur.create(toUtilisateurParams(command as Required<InviterUnUtilisateurCommand>))
-      : utilisateurCourant.duMemeRole(toUtilisateurDuMemeRoleParams(command))
+      ? Utilisateur.create(this.#toUtilisateurParams(command as Required<InviterUnUtilisateurCommand>))
+      : utilisateurCourant.duMemeRole(this.#toUtilisateurDuMemeRoleParams(command))
     if (!utilisateurCourant.peutGerer(utilisateurACreer)) {
       return 'KO'
     }
     const isUtilisateurCreated = await this.#repository.add(utilisateurACreer)
     return isUtilisateurCreated ? 'OK' : 'emailExistant'
   }
+
+  #toUtilisateurParams(
+    command: Required<InviterUnUtilisateurCommand>
+  ): UtilisateurCreateParam {
+    return {
+      ...this.#toUtilisateurDuMemeRoleParams(command),
+      organisation: command.role.organisation,
+      role: command.role.type,
+    }
+  }
+
+  #toUtilisateurDuMemeRoleParams(
+    command: InviterUnUtilisateurCommand
+  ): UtilisateurDuMemeRoleParam {
+    return {
+      derniereConnexion: null,
+      email: command.email,
+      inviteLe: this.#date,
+      isSuperAdmin: false,
+      nom: command.nom,
+      prenom: command.prenom,
+      uid: command.email,
+    }
+  }
```

Comment:
Cette fonction n'aurait pas dû être déplacée : elle ne dépend pas du contexte de l'instance (`this`) et n'a donc pas à être une méthode d'instance. Fonction _top level_ comme avant ou méthode statique.